docs/load_tests/quick_tests/2022-03-22/Load-test-2022-03-22.md

```diff
@@ -0,0 +1,109 @@
+
+## Results
+
+![participants](resources/participants_graph.png)
+
+2'15 after the scheduling of a new JVB pod, Scaleway scheduled a node, appearing with a `Not Ready` state.
+30 seconds later, the node was `Ready`, and we had to wait 15 more seconds for the JVB pod to be in `Pending` state.
+The pod took approximatively 1' to be in `Ready` state.
+
+It took ~4' after the HPA updated the number of desired JVB pods to have a second working JVB pod.
+
+### JVB metrics
+
+| METRIC                | FIRST JVB                                                   | SECOND JVB
+| --------------------- | ----------------------------------------------------------- | --------------------------
+| Received bandwidth    | ![recv bandwidth jvb 1](resources/bandwidth_rcv_jvb_1.png)  | ![recv bandwidth jvb 2](resources/bandwidth_rcv_jvb_2.png)
+| Transmitted bandwidth | ![tx bandwidth jvb 1](resources/bandwidth_tx_jvb_1.png)     | ![tx bandwidth jvb 2](resources/bandwidth_tx_jvb_2.png)
+| Received packets      | ![recv packets jvb 1](resources/packets_rcv_jvb_1.png)      | ![recv packets jvb 2](resources/packets_rcv_jvb_2.png)
+| Transmitted packets   | ![tx packets jvb 1](resources/packets_tx_jvb_1.png)         | ![tx packets jvb 2](resources/packets_tx_jvb_2.png)
+| CPU usage             | ![cpu usage jvb 1](resources/cpu_usage_jvb_1.png)           | ![cpu usage jvb 2](resources/cpu_usage_jvb_2.png)
+| CPU throttling        | ![cpu throttling jvb 1](resources/cpu_throttling_jvb_1.png) | ![cpu throttling jvb 2](resources/cpu_throttling_jvb_2.png)
+| Memory usage          | ![memory usage jvb 1](resources/memory_usage_jvb_1.png)     | ![memory usage jvb 2](resources/memory_usage_jvb_2.png)
+
+### Metrics of other components
+
+| METRIC                   | CHART
+| ------------------------ | -------------------------------------------------------------------
+| Jicofo CPU usage         | ![jicofo cpu usage](resources/jicofo_cpu_usage.png)
+| Prosody CPU usage        | ![prosody cpu usage](resources/prosody_cpu_usage.png)
+| Prosody memory usage     | ![prosody memory usage](resources/prosody_memory_usage.png)
+| Jitsi-front CPU usage    | ![jitsi front cpu usage](resources/jitsi_front_cpu_usage.png)
+| Jitsi-front memory usage | ![jitsi front memory usage](resources/jitsi_front_memory_usage.png)
+
+---
+
+## Interpretation of results
+
+At 15:35:30, the second JVB pod is ready, but there is no load balancing when the JVB is ready.
+This is explained by the fact that participants have already joined their conferences, and were assigned to the first JVB.
+The human participant was assigned to the second JVB because he joined his meeting when the second JVB was ready.
+
+---
+
+## Conclusion
+
+The HPA is working and we need approximatively 5 minutes to have a second JVB, further investigations will be needed to optimize the scaling of JVB pods.
+
+Scale in advance is a real issue because once a participant has joined a conference, he can not be moved to another JVB.
```

Comment:
I didn't get this sentence. What the link between "scaling in advance" and "moving to another JVB"?

Reply:
I added some details, let me know if it is still unclear !